label_studio/core/management/commands/locked_migrate.py

```diff
@@ -33,10 +32,17 @@ def handle(self, *args, **options):
 
         lock_id = options['migrate_lock_id']
         with connection.cursor() as cursor:
+            logger.info(f'Attempting to acquire a migration lock: {lock_id}')
+            cursor.execute(f'SELECT pg_advisory_lock({lock_id})')
+            logger.info('Successfully acquired the lock. Starting migrations...')
             try:
-                logger.info(f'Attempting to acquire a migration lock: {lock_id}')
-                cursor.execute(f'SELECT pg_advisory_xact_lock({lock_id})')
-                logger.info('Successfully acquired the lock. Starting migrations...')
                 MigrateCommand.handle(self, *args, **options)
+            except DatabaseError as migration_error:
+                logger.error(f'Error during migration execution: {migration_error}')
+                raise
```

Comment:
ah I realize I misunderstood the `try...finally` approach in the original version, I don't think this is needed and `try...finally` without any `except` block here or below is fine.